### ISSUE TITLE
loss_ssim is not provided?

### ISSUE THREAD

**Issue by lizs17** [ayanglab/SwinMR#7]
It seems loss_ssim.py is missing in the model/ directory ....

**Comment by fsa125** [comment]
Yes, the loss_ssim.py is not provided by the authors. However, you can comment the portion and run the code. It works.

**Comment by aayshhh** [comment]
for me when running the program a nan error is showing on conv_last
